server/app/views/components/breadcrumb/BreadcrumbItem.java

```diff
@@ -0,0 +1,38 @@
+package views.components.breadcrumb;
+
+import com.google.auto.value.AutoValue;
+import javax.annotation.Nullable;
+import views.components.Icons;
+
+/** Represents a single item in a breadcrumb list. */
+@AutoValue
+public abstract class BreadcrumbItem {
+  public static BreadcrumbItem create(
```

Comment:
**question (blocking):** With this API, it seems like it would be possible to assemble a list of incompatible 'BreadcrumbItem`s. For instance, `isCurrentPage` is `true` for more than one item (or zero items, or exactly one item that is not the last in the list).

Am I understanding this correctly? If so, can the API be revised to prevent such occurrences? Perhaps infer that the last item is the current one?

Reply:
Oh, now I see there is an exception for one of these cases. But if it's possible to prevent this in the first place, that seems preferable.

Reply:
Updated to always put `aria-current="page"` on the last breadcrumb. Thanks!